doc/source/whatsnew/v2.1.0.rst

```diff
@@ -372,10 +372,10 @@ I/O
 ^^^
 - :meth:`DataFrame.to_orc` now raising ``ValueError`` when non-default :class:`Index` is given (:issue:`51828`)
 - :meth:`DataFrame.to_sql` now raising ``ValueError`` when the name param is left empty while using SQLAlchemy to connect (:issue:`52675`)
+- Bug in :func:`json_normalize`, json_normalize cannot parse metadata fields list type (:issue:`#37782`)
```

Comment:
```suggestion
- Bug in :func:`json_normalize`, json_normalize cannot parse metadata fields list type (:issue:`37782`)
```

Reply:
@mroeschke I'm seeing the message: This branch has conflicts that must be resolved
Only those with [write access](https://docs.github.com/articles/what-are-the-different-access-permissions) to this repository can merge pull requests.
Conflicting files
doc/source/whatsnew/v2.1.0.rst

Can you please take a look and guide me on this?

Reply:
Please follow these constructions to update your branch and fix the conflict: https://pandas.pydata.org/docs/development/contributing.html#updating-your-pull-request

Reply:
@mroeschke is there anything else that I have to do? I only see a "Close pull request" button at bottom of the page. This should be pressed?? (Sorry, just afraid to mess anything else!)